appium/tests/specs/mock/setup/CheckRefreshKeyFromEKMSecond.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { MockApi } from 'api-mocks/mock';
+import {
+  KeysScreen,
+  MenuBarScreen,
+  SettingsScreen,
+  SetupKeyScreen,
+  SplashScreen,
+} from '../../../screenobjects/all-screens';
+import { ekmPrivateKeySamples } from "../../../../api-mocks/apis/ekm/ekm-endpoints";
+import { CommonData } from "../../../data";
+import BaseScreen from "../../../screenobjects/base.screen";
+import AppiumHelper from "../../../helpers/AppiumHelper";
+
+const goToKeysScreen = async () => {
+  await MenuBarScreen.clickMenuIcon();
+  await MenuBarScreen.clickSettingsButton();
+  await SettingsScreen.clickOnSettingItem('Keys');
+}
```

Comment:
Since this is written three times, you could add it as a method to `KeysScreen`. Or alternatively to `InboxScreen` if there is one.